project/website/src/global.mts

```diff
@@ -0,0 +1,36 @@
+// @ts-ignore
+import hljs from 'https://unpkg.com/@highlightjs/cdn-assets@11.11.1/es/highlight.min.js';
```

Comment:
We don't support bundling yet in the static site builder, which is why this is a raw URL reference